tests/cleanup/remove_old_vms_aws.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# Script removes virtual machines older than HOURS_LIMIT (24 hours by default) from Amazon EC2
+
+. /usr/share/beakerlib/beakerlib.sh
+
+
+rlJournalStart
+    rlPhaseStartSetup
+        if [ -z "$AWS_ACCESS_KEY_ID" ]; then
+            rlFail "AWS_ACCESS_KEY_ID is empty!"
+        else
+            rlLogInfo "AWS_ACCESS_KEY_ID is configured"
+        fi
+
+        if [ -z "$AWS_SECRET_ACCESS_KEY" ]; then
+            rlFail "AWS_SECRET_ACCESS_KEY is empty!"
+        else
+            rlLogInfo "AWS_SECRET_ACCESS_KEY is configured"
+        fi
+
+        # VMs older than HOURS_LIMIT will be deleted
+        HOURS_LIMIT="${HOURS_LIMIT:-24}"
+        export TIMESTAMP=`date -u -d "$HOURS_LIMIT hours ago" '+%FT%T'`
+        export AWS_REGION="${AWS_REGION:-us-east-1}"
+
+        rlLogInfo "HOURS_LIMIT=$HOURS_LIMIT"
```

Comment:
will probably make sense to log the TIMESTAMP as well. If we need to debug something we can see the local timestamp from the machine running this script and then compare with timestamps logged from Ansible.

Reply:
Yes, it can be logged here. On the other hand, TIMESTAMP is printed later in case you need it:
rlLogInfo "Removing VMs created before $TIMESTAMP"